events/sktime-onboarding-days.md

```diff
@@ -1,14 +1,14 @@
 # sktime onboarding days
-Data scientists with an interest in contributing to sktime are invited to join the sktime onboarding days with new joiners, summer programme participants, and core contributors, on May 19 – 21.
+Data scientists with an interest in contributing to sktime are invited to join the sktime onboarding days with new joiners, summer programme participants, and core contributors, on May 25 – 27.
 
-[Register now!](https://www.eventbrite.com/e/sktime-onboarding-days-tickets-154010598847)
+[TO BE UPDATED: Register via eventbrite](https://www.eventbrite.com/e/sktime-onboarding-days-tickets-154010598847)
```

Comment:
I assume this link is only for last year? If so, then I would recommend removing the link and saying a link will be added